LongWarOfTheChosen/Src/LW_PerkPack_Integrated/Classes/X2Effect_Immobilize.uc

```diff
@@ -10,7 +10,16 @@ simulated protected function OnEffectAdded(const out EffectAppliedData ApplyEffe
 	local XComGameState_Unit UnitState;
 
 	UnitState = XComGameState_Unit(kNewTargetState);
-	UnitState.SetUnitFloatValue(class'X2Ability_DefaultAbilitySet'.default.ImmobilizedValueName, 1);
+
+	if(UnitState.IsChosen())
+	{
+		AddPersistentStatChange(eStat_Mobility, 0.5f, MODOP_Multiplication);
```

Comment:
Something we need to consider is that MODOP_Multiplication comes before any MODOP_Addition, which means that any flat bonus to Mobility will allow the unit to move. I think MODOP_PostMultiplication is more appropriate (and is what Viper's "Bound" status effect uses).